### PR TITLE
Add covid monitoring blogpost

### PR DESCRIPTION
**Pull request by simonbray**
ping @bgruening @wm75

### CODE REVIEW

**Comment by beatrizserrano** [comment]
@simonbray I will work on a figure to illustrate this

**Comment by simonbray** [comment]
If that image is finished, can we upload it and merge? @wm75 did you have any other comments?

**Comment by bgruening** [comment]
Title proposed, ok?

**Comment by beatrizserrano** [comment]
I need to make minor changes to the figure, and export a light version

File: _posts/2021-04-28-sars-cov-2-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+* variants in VCF format
+* alignment files in BAM format
+* consensus sequences in FASTA format
+
+These are subsequently pushed back to a public FTP server hosted by [CRG/BSC](ftp://xfer13.crg.eu). CRG then takes this data and
+integrates it into the [Viral Beacon](https://covid19beacon.crg.eu/).
+
+The scripts for the workflow automation are [publicly available](https://github.com/usegalaxy-eu/ena-cog-uk-wfs) together with [documentation](https://github.com/usegalaxy-eu/ena-cog-uk-wfs/blob/main/docs/manual.md). The analysis is based on [public, validated workflows](https://www.biorxiv.org/content/10.1101/2021.03.25.437046v1) that are available via [workflowhub.eu](https://workflowhub.eu/).
+
+Special care was taken to run this analysis on scale without disrupting normal Galaxy usage. In other words, we can monitor samples, while at the same time still run trainings and serve hundreds of other Galaxy users simultaneously.
```
Comment (wm75) [inline _posts/2021-04-28-sars-cov-2-monitoring.md:29]:
```suggestion
Special care was taken to run this analysis on scale without disrupting normal Galaxy usage. In other words, we can monitor samples, and at the same time still run trainings and serve hundreds of other Galaxy users simultaneously.
```

File: _posts/2021-04-28-sars-cov-2-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+If you want to process samples from your country, please get in touch with us and we will help you get things running!
+
+
+## Request processing of your own samples
+
+In addition, we wanted to make this resource useful for everyone, including non-Galaxy users. Therefore, we have opened up contribution via a simple GitHub repository: [https://github.com/usegalaxy-eu/sars-cov-2-processing-requests](https://github.com/usegalaxy-eu/sars-cov-2-processing-requests). The idea is very simple; you just submit a file with URLs (ENA URLs for example) as a single file. This file makes up one batch of processing requests.
+
+One of the European Galaxy Team will check your contribution and approve it, after which Galaxy will pick those files up and process them with our standard pipelines.
```
Comment (wm75) [inline _posts/2021-04-28-sars-cov-2-monitoring.md:38]:
```suggestion
Members of the European Galaxy Team will check your contribution and approve it, after which Galaxy will pick those files up and process them with our standard pipelines.
```

File: _posts/2021-04-28-sars-cov-2-monitoring.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+Just as for the automated COG-UK data, all files are pushed back to the CRG FTP server. So in the end, after a few hours, (depending on the amount of URLs and the load on our servers) the user can either
```
Comment (wm75) [inline _posts/2021-04-28-sars-cov-2-monitoring.md:40]:
```suggestion
Just as for the automated COG-UK data analysis, all files are pushed back to the CRG FTP server. So in the end, after a few hours (depending on the amount of URLs and the load on our servers), you can access the results through either:
```

File: _posts/2021-04-28-sars-cov-2-monitoring.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+get the results from:
```
Comment (wm75) [inline _posts/2021-04-28-sars-cov-2-monitoring.md:41]:
```suggestion
```

File: _posts/2021-04-28-sars-cov-2-monitoring.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+* Galaxy via shared/public histories
+* via the CRG FTP server
+
+and visualize them on the [Viral Beacon](https://covid19beacon.crg.eu) (probably a few days later).
```
Comment (wm75) [inline _posts/2021-04-28-sars-cov-2-monitoring.md:46]:
```suggestion
and see them visualized on the [Viral Beacon](https://covid19beacon.crg.eu) dashboard (with a delay of probably a few days).
```

File: _posts/2021-04-28-sars-cov-2-monitoring.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+We think this makes it even easier to analyze virus sequences for people that don't want to run their own workflows,
```
Comment (wm75) [inline _posts/2021-04-28-sars-cov-2-monitoring.md:48]:
```suggestion
This should make it even easier to analyze viral sequences with Galaxy, in particular, for people who don't want to run their own workflows,
```

File: _posts/2021-04-28-sars-cov-2-monitoring.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
+---
+site: freiburg
+tags: [data, COVID-19]
+title: SARS-CoV-2 monitoring of raw open data for disease surveillance
```
Comment (wm75) [inline _posts/2021-04-28-sars-cov-2-monitoring.md:4]:
```suggestion
title: Monitoring of raw open data for SARS-CoV-2 genome surveillance
```

File: _posts/2021-04-28-sars-cov-2-monitoring.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+supporters:
+- denbi
+- elixir
+---
+
+Recently, we started a large-scale project on the European Galaxy server for automatically collecting,
+processing and publishing COG-UK and other SARS-CoV-2 sequencing data. We aim to make all these steps fully open and accessible,
```
Comment (wm75) [inline _posts/2021-04-28-sars-cov-2-monitoring.md:11]:
```suggestion
and analyzing raw sequencing data published by COG-UK and, potentially, other large-scale SARS-CoV-2 sequencing projects. As a unique feature, we aim to make all analysis steps leading from sequenced reads to lists of genomic variants and consensus sequences fully open and accessible.
```

File: _posts/2021-04-28-sars-cov-2-monitoring.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+so you too can contribute data, wait for it to be processed, and visualize the results.
+
+
+## Automated variant calling workflows
+
+Every three hours, newly published COG-UK samples are processed to produce:
+
+* a public Galaxy history, which provides full data provenance
```
Comment (wm75) [inline _posts/2021-04-28-sars-cov-2-monitoring.md:19]:
```suggestion
* public Galaxy histories, which provide full data provenance
```

File: _posts/2021-04-28-sars-cov-2-monitoring.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+tags: [data, COVID-19]
+title: SARS-CoV-2 monitoring of raw open data for disease surveillance
+supporters:
+- denbi
+- elixir
+---
+
+Recently, we started a large-scale project on the European Galaxy server for automatically collecting,
+processing and publishing COG-UK and other SARS-CoV-2 sequencing data. We aim to make all these steps fully open and accessible,
+so you too can contribute data, wait for it to be processed, and visualize the results.
```
Comment (wm75) [inline _posts/2021-04-28-sars-cov-2-monitoring.md:12]:
```suggestion
```

File: _posts/2021-04-28-sars-cov-2-monitoring.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+* consensus sequences in FASTA format
+
+These are subsequently pushed back to a public FTP server hosted by [CRG/BSC](ftp://xfer13.crg.eu). CRG then takes this data and
+integrates it into the [Viral Beacon](https://covid19beacon.crg.eu/).
+
+The scripts for the workflow automation are [publicly available](https://github.com/usegalaxy-eu/ena-cog-uk-wfs) together with [documentation](https://github.com/usegalaxy-eu/ena-cog-uk-wfs/blob/main/docs/manual.md). The analysis is based on [public, validated workflows](https://www.biorxiv.org/content/10.1101/2021.03.25.437046v1) that are available via [workflowhub.eu](https://workflowhub.eu/).
+
+Special care was taken to run this analysis on scale without disrupting normal Galaxy usage. In other words, we can monitor samples, while at the same time still run trainings and serve hundreds of other Galaxy users simultaneously.
+
+If you want to process samples from your country, please get in touch with us and we will help you get things running!
```
Comment (wm75) [inline _posts/2021-04-28-sars-cov-2-monitoring.md:31]:
```suggestion
If you want us to process samples from your country, too, please get in touch with us and we will help you get things running!
```

File: _posts/2021-04-28-sars-cov-2-monitoring.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+
+
+## Request processing of your own samples
```
Comment (wm75) [inline _posts/2021-04-28-sars-cov-2-monitoring.md:34]:
```suggestion
## Request processing of your samples of interest
```

File: _posts/2021-04-28-sars-cov-2-monitoring.md [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+
+In addition, we wanted to make this resource useful for everyone, including non-Galaxy users. Therefore, we have opened up contribution via a simple GitHub repository: [https://github.com/usegalaxy-eu/sars-cov-2-processing-requests](https://github.com/usegalaxy-eu/sars-cov-2-processing-requests). The idea is very simple; you just submit a file with URLs (ENA URLs for example) as a single file. This file makes up one batch of processing requests.
```
Comment (wm75) [inline _posts/2021-04-28-sars-cov-2-monitoring.md:36]:
```suggestion
In addition, we wanted to make this automated resource useful for everyone, including non-Galaxy users. To this end, we have opened up an on-demand analysis service via a simple GitHub repository: [https://github.com/usegalaxy-eu/sars-cov-2-processing-requests](https://github.com/usegalaxy-eu/sars-cov-2-processing-requests). The idea is very simple; you just submit a file with URLs (ENA URLs for example) as a single file. This file makes up one batch of processing requests.
```

File: _posts/2021-04-28-sars-cov-2-monitoring.md [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+Just as for the automated COG-UK data, all files are pushed back to the CRG FTP server. So in the end, after a few hours, (depending on the amount of URLs and the load on our servers) the user can either
+get the results from:
+
+* Galaxy via shared/public histories
+* via the CRG FTP server
+
+and visualize them on the [Viral Beacon](https://covid19beacon.crg.eu) (probably a few days later).
+
+We think this makes it even easier to analyze virus sequences for people that don't want to run their own workflows,
+but are still interested in specific samples.
```
Comment (wm75) [inline _posts/2021-04-28-sars-cov-2-monitoring.md:49]:
```suggestion
but are still interested in specific batches of samples not covered by our automated monitoring efforts.
```

File: _posts/2021-04-28-sars-cov-2-monitoring.md [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+
+
+## Interactive ObservableHQ dashboard
```
Comment (wm75) [inline _posts/2021-04-28-sars-cov-2-monitoring.md:52]: Move this up above the user-requested analysis section.

**Review by wm75 — COMMENTED** [review]
Last minute comments.

**Comment by bgruening** [comment]
Thanks everyone!

**Pull request merged**